tests/assets/db/general/suites/default/index.yaml

```diff
@@ -1,7 +1,5 @@
 description: 'layout of ltplite for KPET'
-patterns:
-    -   pattern: .*
-        case_name: default/ltplite
+maintainer: maint1
+fetch_suffix: distribution/ltp/lite
 cases:
-    -   name: default/ltplite
-        tasks: suites/default/ltplite.xml
+    -   name: /kernel/distribution/ltp/lite
```

Comment:
Please make this case name change a separate commit.

Reply:
The generated task name in XML doesn't actually change. The original has 
<task name="/kernel/distribution/ltp/lite" role="STANDALONE"> , so this change is here to make sure the output remains the same. Are you sure you want to split this?

Reply:
Ah, if I got it right, and this is just bringing in the task's name, like all those changes in kpet-db, then sure, please keep it! Thanks :)